plugins/arDominionB5Plugin/modules/repository/templates/_uploadLimit.php

```diff
@@ -18,11 +18,15 @@
   </h5>
 
   <div class="card-body py-2">
+    <style <?php echo __(sfConfig::get('csp_nonce', '')); ?>>
+	#upload-limit-progress-div { height : 25px }
```

Comment:
This one could be changed to BS5 type

Reply:
BS doesn't have specfic classes non percentage sizing like px so will leave as is